pytwis/pytwis_clt.py

```diff
@@ -319,20 +324,20 @@ def pytwis_command_parser(raw_command):
             args = splited_raw_command[1] 
             arg_dict = parse.parse('{{{arg1}}} {{{arg2}:d}}'.format(arg1=CmdConstant.ARG_USERNAME,
                                                                     arg2=CmdConstant.ARG_MAX_TWEETS), 
-                                   args)
+                                                                    args)
             if arg_dict is None:
```

Comment:
If arg_dict is not None, do we need to set "arg_dict = arg_dict.named"?

Reply:
I changed the code to check if arg_dict is parse.Result and do the conversion at the end.